exonum-java-binding-core/rust/src/runtime/cmd.rs

```diff
@@ -1,97 +1,53 @@
 const EJB_SERVICE_CLASSPATH: &str = "EJB_SERVICE_CLASSPATH";
-const EJB_LIBPATH: &str = "EJB_LIBPATH";
 const EJB_MODULE_NAME: &str = "EJB_MODULE_NAME";
 const EJB_PORT: &str = "EJB_PORT";
-const EJB_JVM_CONFIG_NAME: &str = "ejb_jvm_config";
+const EJB_PUBLIC_CONFIG_NAME: &str = "ejb_public_config";
 pub const EJB_CONFIG_NAME: &str = "ejb";
 
 pub struct GenerateNodeConfig;
 
 impl CommandExtension for GenerateNodeConfig {
     fn args(&self) -> Vec<Argument> {
-        vec![
-            Argument::new_named(
-                EJB_JVM_ARGUMENTS,
-                false,
-                "Additional parameters for JVM. Must not have a leading dash. \
-                 For example, `Xmx2G` or `Xdebug`",
-                None,
-                "ejb-jvm-args",
-                true,
-            ),
-            Argument::new_named(
-                EJB_LOG_CONFIG_PATH,
-                false,
-                "Path to log4j configuration file.",
-                None,
-                "ejb-log-config-path",
-                false,
-            ),
-            Argument::new_named(
-                EJB_SYSTEM_CLASSPATH,
-                true,
-                "Java bindings framework system classpath.",
-                None,
-                "ejb-classpath",
-                false,
-            ),
-            Argument::new_named(
-                EJB_SERVICE_CLASSPATH,
-                true,
-                "Java service classpath.",
-                None,
-                "ejb-service-classpath",
-                false,
-            ),
-            Argument::new_named(
-                EJB_LIBPATH,
-                true,
-                "Path to java-bindings shared library.",
-                None,
-                "ejb-libpath",
-                false,
-            ),
-        ]
+        vec![Argument::new_named(
+            EJB_MODULE_NAME,
+            true,
+            "A fully-qualified class name of the user service module.",
+            None,
+            "ejb-module-name",
+            false,
+        )]
     }
 
     fn execute(&self, mut context: Context) -> Result<Context, failure::Error> {
-        let user_parameters = context.arg_multiple(EJB_JVM_ARGUMENTS).unwrap_or_default();
-        let log_config_path = context.arg(EJB_LOG_CONFIG_PATH).unwrap_or_default();
-        let system_class_path = context.arg(EJB_SYSTEM_CLASSPATH)?;
-        let service_class_path = context.arg(EJB_SERVICE_CLASSPATH)?;
-        let lib_path = context.arg(EJB_LIBPATH)?;
+        let module_name = context.arg(EJB_MODULE_NAME)?;
 
-        let jvm_config = JvmConfig {
-            user_parameters,
-            system_class_path,
-            service_class_path,
-            lib_path,
-            log_config_path,
-        };
+        let public_config = PublicConfig { module_name };
 
-        let mut services_secret_configs = context
-            .get(keys::SERVICES_SECRET_CONFIGS)
+        let mut services_public_configs = context
```

Comment:
I'd consider splitting into methods or at least sections with comments clarifying the steps.

I assume it is 
// Create a public config of this service and node? — BTW, why each node has to specify module name? Shan't it be global config?

// Put this service config at this node — But how at this stage can we have several of them?